src/ui/components/shared/WorkspaceSettingsModal/WorkspaceSettingsModal.tsx

```diff
@@ -61,21 +64,28 @@ export function WorkspaceMembers({
   );
 }
 
-type WorkspaceFormProps = {
-  members?: WorkspaceUser[];
-};
-
-function WorkspaceForm({ members = [] }: WorkspaceFormProps) {
+function WorkspaceForm() {
   const workspaceId = useGetTeamIdFromRoute();
   const [inputValue, setInputValue] = useState("");
   const [errorMessage, setErrorMessage] = useState<string | null>(null);
   const [isLoading, setIsLoading] = useState(false);
-  const inviteNewWorkspaceMember = hooks.useInviteNewWorkspaceMember(() => {
-    setInputValue("");
-    setIsLoading(false);
-  });
+  const inviteNewWorkspaceMember = hooks.useInviteNewWorkspaceMember(
+    () => {
+      setInputValue("");
+      setIsLoading(false);
+    },
+    err => {
+      trackEvent("error.add_member_error");
+      if (getBackendErrorMessage(err) === USER_ALREADY_INVITED_MESSAGE) {
+        setErrorMessage(USER_ALREADY_INVITED_MESSAGE);
```

Comment:
nit: "User has already been invited" isn't a super user-friendly message for to display.

Maybe we can say something more like the other case, e.g. "This team member already has a pending invitation."

Reply:
I updated the error message